frontend/src/components/VolunteerInfo.js

```diff
@@ -0,0 +1,211 @@
+  			this.setState({notCheckedBox: true})
+  		}
+  		else {
+			//make sure this works correctly!
+			this.addDays();
+
+			console.log(this.state.personalData)
+			this.sendVolunteerInfo(this.state.personalData)
+  		}
+
+		event.preventDefault();
+	}
+
+	sendVolunteerInfo = (info) => {
+		let _this = this
+        fetch(env.backendURL + 'volunteers/updateVolunteerInfo', {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(info)
+        })
+        .then((res) => {
+            if (res.status === 404) {
+                _this.setState({error: true})
+            }
+            else {
+                _this.props.history.push("/sitemanager");
+            }
+        })
+	}
+
+	//to do:
+	// - display page if not filled by user 
+	render(){
+
+		const { Monday, Tuesday, Wednesday, Thursday, Friday } = this.state;
+
+		return(
+			<div className='VolunteerInfo-form'>
+				<h2 id="header"> Volunteer Additional Information </h2>
+				
+				<form onSubmit={this.addInfo}>
+					<label for="phoneNumber"> Phone Number:* </label>
+					<br/>
+					<input type="text" id="phoneNumber" placeholder='Phone Number' onChange={this.updateInfo} size="12" required/> 
+					
+					<br/>
+					<label for="email"> Email:* </label>
+					<br/>
+					<input type="email" id="email" placeholder="Email ex: example@gmail.com" onChange={this.updateInfo} size="50" required/>
+
+					<br/>
+					<div className="days-display">
+						<label id="days-text">Select Days Available:* </label>
+
+						<div className="days-row">
+
+							<label id="checkbox-display"> 
+								<input name="Monday"  
+								type="checkbox" 
+								checked={Monday} 
+								onChange={this.handleChange} />
+							 	Monday 
+							</label>
+
+							<label id="checkbox-display"> 
+								<input name="Tuesday"  
+								type="checkbox" 
+								checked={Tuesday} 
+								onChange={this.handleChange} />
+							 	Tuesday 
+							</label>
+
+							<label id="checkbox-display"> 
+								<input name="Wednesday"  
+								type="checkbox" 
+								checked={Wednesday} 
+								onChange={this.handleChange} />
+							 	Wednesday 
+							</label>
+
+							<label id="checkbox-display"> 
+								<input name="Thursday"  
+								type="checkbox" 
+								checked={Thursday} 
+								onChange={this.handleChange} />
+							 	Thursday 
+							</label>
+
+							<label id="checkbox-display"> 
+								<input name="Friday"  
+								type="checkbox" 
+								checked={Friday} 
+								onChange={this.handleChange} />
+							 	Friday 
+							</label>
+
+
+
+						</div>
+						{this.state.notCheckedBox && <div className="checkbox-error">Select one of more days</div>}
+					</div>
+
+					<div className="comments">
+						<label> Comments  (not required): </label>
+						<br/>
+						<textarea
+						id="notes"
+						name="comments"
+						value={this.state.comments}
```

Comment:
Make the default comment value an empty string, and set the placeholder value as the message